### ISSUE TITLE
Plugin Apps

### ISSUE THREAD

**Issue by dennisreimann** [btcpayserver/btcpayserver-doc#1252]
The app functionality for plugins needs documentation and potentially an addition to the template plugin.

**Comment by pavlenex** [comment]
I don't understand this issue, what's an app functionality for plugins, is this still relevant?

**Comment by dennisreimann** [comment]
Plugins can implement the [`AppType`](https://github.com/btcpayserver/btcpayserver/blob/master/BTCPayServer/Services/Apps/AppType.cs) interface, which comes with special traits: Like the POS or Crowdfund those "apps" (we still have that legacy term in the code and will most likely keep it there) can e.g. be mounted to domains and show data on the dashboard.

**Comment by pavlenex** [comment]
Thanks, would it still be relevant with this https://github.com/btcpayserver/btcpayserver/issues/5588 ?

**Comment by dennisreimann** [comment]
Yes, because external plugins use it as well, e.g. PodServer and Shoutout.
